projects/imagemagick/run_tests.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Usage: bash run_test.sh 
+# Runs all tests.
+
+set -e # Exit immediately if any command fails
+
+PROJECT_SRC_DIR="$SRC/imagemagick"
+
+echo "==> Changing to ImageMagick source directory: $PROJECT_SRC_DIR"
+cd "$PROJECT_SRC_DIR"
+
+echo "==> Setup needed right now..."  # DO NOT SUBMIT
+./configure
+echo "==> Setup finished successfully."  # DO NOT SUBMIT
+
+echo "==> Running all tests..."
+make check
```

Comment:
there's also `make check-TESTS`. do you know what the difference is?

I found that you can also pass `-j$(nproc)` to make things a bit faster.

Reply:
Answered bellow. Thanks for the suggestions.